SPHINXsys/src/shared/geometries/level_set.cpp

```diff
@@ -17,17 +17,25 @@ namespace SPH
 		}
 	}
 	//=================================================================================================//
-	Real BaseLevelSet::computeHeaviside(Real phi, Real half_width)
+	Real BaseLevelSet::Heaviside(Real phi, Real half_width)
 	{
 		Real heaviside = 0.0;
 		Real normalized_phi = phi / half_width;
 		if (phi < half_width && phi > -half_width)
-			heaviside = (0.5 + 0.5 * normalized_phi) + 0.5 * sin(Pi * normalized_phi) / Pi;
+			heaviside = 0.5 + 0.5 * normalized_phi;
 		if (normalized_phi > 1.0)
 			heaviside = 1.0;
 		return heaviside;
```

Comment:
```suggestion
		Real normalized_phi = phi / half_width;
		return std::clamp(0.5 + 0.5 * normalized_phi, 0.0, 1.0);
```

Reply:
The function is moved to scalar_function.cpp as you suggested.